src/ms.tex

```diff
@@ -205,52 +221,92 @@ \subsection{Contributor base} \label{sec:project-contributors}
 % Transition of needs: From new features, to community development, sustainability, ...
 % - pipeline between User, Participant, Code contributor, Maintainer, Coordinator.
 
+\subsection{Inclusion, Diversity, and Equity Programs} \label{sec:project-ide}
+
+\secauthor{Lía Corrales}
 
 
 \subsection{Funding} \label{sec:project-funding}
 
-New funding sources.
-% Funding received:
-% - Moore
-% - NASA
+\secauthor{Aarya Patil}
```

Comment:
% Astronomy Open Development built on-top of Open Blockchain Protocol Standards (perhaps subsubsection)

Reply:
@sebaguro thanks for the idea here, but as I mentioned in my email, I think this is out of scope for this paper.